app/core/logger.ts

```diff
@@ -55,7 +55,7 @@ export class Logger {
 
         global && this.global();
 
-        if ( store || this._loadLevel() ) this.store();
+        if ( store ) this.store();
```

Comment:
if store is false it will call _loadLevel() which will try to access window.localStorage which will break when doing server side rendering using Angular Universal.